test/controllers/orders_controller_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
+
+class OrdersControllerTest < ActionDispatch::IntegrationTest
+
+def new
+  @order = Order.new
+  authorize @order
```

Comment:
We won't use Pundit, so we will not add any authorize in the code

Reply:
done